default.nix

```diff
@@ -59,10 +69,17 @@
     }
 
 , mkDerivation   ? null
+
+# **** 
```

Comment:
Does this mark the end of the section?

Reply:
I explained it in the main Report.

In Haskell Haddock&outshine ideal togather.

Nix has `{..}` and brackets, so structure of `}` should be preserved. So user does not move it around with the last subheading block - the `}` is last subheading block in itself. And first `{` goes into the main heading block - that allows robustness of the structure to the movements and restructures.

Reply:
Thanks, I remember now :)